internal/storage/writer/talaria/README.md

```diff
@@ -0,0 +1,15 @@
+# Talaria
+
+This sink implements sending data to a second Talaria. It can be used when there is one Talaria for **event ingestion**, and needs to write data to a second Talaria, which is used **purely for querying data.**
+
+This sink can be enabled by adding the following configuration in the `storage` section:
+
+```yaml
+storage:
+  compact:                               # enable compaction
+    interval: 60                         # compact every 60 seconds
+    nameFunc: "s3://bucket/namefunc.lua" # file name function
+    talaria:                             # sink to Talaria
+      endpoint: "127.0.0.1:8043"         # Talaria endpoint to write data to
```

Comment:
Add more options as referred in the https://github.com/kelindar/talaria/blob/master/client/golang/client_opts.go